USFMToolsSharp.Renderers.HTML/HtmlRenderer.cs

```diff
@@ -252,9 +264,10 @@ private string RenderMarker(Marker input)
                     output.AppendLine("</span>");
                     break;
                 case HMarker hMarker:
-                    output.AppendLine("<div class=\"header\">");
+                    output.AppendLine(string.Format("<div class=\"header\" id=\"{0}\">", hMarker.HeaderText));

```

Comment:
You could be looking at TOC 1,2,3 markers instead of H

Reply:
And having the header text as the Id I think is incorrect because you could have two headers with the same value in theory.